src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,19 @@
+import { UserInfo } from '../UserInfo';
+
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+export const TodoInfo = ({ todo }) => {
+  function getClassName() {
+    if (todo.completed === true) {
+      return 'TodoInfo--completed';
+    }
+
+    return ' ';
```

Comment:
why `' '` instead of empty string?
anyway - use classnames library - it's already installed